.github/workflows/test.yaml

```diff
@@ -168,6 +170,7 @@ jobs:
       - name: Run unit tests
         env:
           _image: ${{ steps.build.outputs.id }}
+          NUTILS_DEBUG: all
```

Comment:
I don't think this works. You can set environment variables inside the container with `podman run ... --env ...`:

```sh
podman run --pull=never --rm -v "$PWD/tests:/app/tests:ro" -v "$PWD/examples:/app/examples:ro" --env NUTILS_DEBUG=all "$_image" -m unittest -bq
```

Reply:
Done. Thanks!